core/lib/zksync_core/src/api_server/web3/tests/mod.rs

```diff
@@ -141,10 +144,13 @@ async fn spawn_server(
     websocket_requests_per_minute_limit: Option<NonZeroU32>,
     tx_executor: MockTransactionExecutor,
     stop_receiver: watch::Receiver<bool>,
+    api_config: Option<InternalApiConfig>,
```

Comment:
Bikeshedding: This looks weird. When we've talked about this, I've meant passing `api_config: InternalApiConfig` to this function and `spawn_http_server` *instead of* `network_config`; sorry for not being clear. `InternalApiConfig` can be constructed on the caller side; it'd lead to some code duplication, but it'd be fairly minor.